alws/crud/build_node.py

```diff
@@ -380,10 +380,29 @@ def append_errata_package(_, errata_package, artifact, rpm_info):
             conditions.append(
                 models.ErrataPackage.arch == rpm_info['arch']
             )
-        errata_packages = await db.execute(select(
-            models.ErrataPackage
-        ).where(sqlalchemy.and_(*conditions)))
-        for errata_package in errata_packages.scalars().all():
+
+        query = select(models.ErrataPackage).where(
+            sqlalchemy.and_(*conditions)
+        )
+
+        if module_index:
+            # good idea?
+            # we do not build more than one module at a time,
+            # do we?
```

Comment:
I suppose we don't need this comment anymore

Reply:
done